images/arch/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM docker.io/archlinux/base:latest
```

I think "archlinux" is more official. It's referenced from https://www.archlinux.org/download/

```suggestion
FROM docker.io/archlinux:latest
```